backend/jest.config.js

```
@@ -0,0 +1,18 @@
+/** @type {import('jest').Config} */
+module.exports = {
```

ESLint

[error] 'module' is not defined.

(no-undef)